app/src/gecko/java/org/mozilla/tv/firefox/ext/View.kt

```diff
@@ -26,6 +30,27 @@ fun View.validateKnownViewById(): Int {
     return this.id
 }
 
+/**
+ * Set the horizontal margins on the given view.
+ *
+ * We want to add padding to the beginning and end of the RecyclerView: ideally we'd just add
+ * paddingStart/End. Unfortunately, this causes a visual glitch as each card scrolls offscreen.
+ * Instead, we set the margins for the first and last card.
+ *
+ * See [DefaultChannelAdapter.onBindViewHolder] for an example
+ */
+fun View.setHorizontalMargins(context: Context, position: Int, itemCount: Int) {
```

Comment:
Does this need to be declared in both System and Gecko?  The implementations seem to be the same, unless I'm missing something.  Would it be possible to merge these?

Reply:
View ext already exists in both variants, moving to main would raise a compiler issue

Reply:
I don't understand: what compiler issue would you see?

Reply:
Nvm, looks like this was removed anyway.

Reply:
@mcomella At any given time, there would have been two files with the same package statement & name (one from main, one from either system or GV).